src/utils/emasser/generateConfig.ts

```diff
@@ -0,0 +1,274 @@
+import path from 'path'
+import colors from 'colors' // eslint-disable-line no-restricted-imports
+import fse from 'fs-extra'
+import dotenv from 'dotenv'
+import inquirer from 'inquirer'
+import inquirerFileTreeSelection from 'inquirer-file-tree-selection-prompt'
+
+const PROMPT_MESSAGE = [
+  'Provide the eMASS API key (api-key) - valid key is > 30 alpha numeric characters:',
```

Comment:
Provide the eMASS API key (EMASS_API_KEY) - valid key is > 30 alpha numeric characters: